Mentioning that `loading` attribute of `img` must be placed before `src` (#26778)
mdn/content: files/en-us/web/html/element/img/index.md

    > **Note:** Loading is only deferred when JavaScript is enabled. This is an anti-tracking measure, because if a user agent supported lazy loading when scripting is disabled, it would still be possible for a site to track a user's approximate scroll position throughout a session, by strategically placing images in a page's markup such that a server can track how many images are requested and when.

>     > **Note:** A [Firefox bug](https://bugzil.la/1647077) requires that the **`loading` attribute must be placed before the `src` attribute**, otherwise it has no effect.

Reviewer: Perhaps we don't need more emphasis since this is already a note

```suggestion
    > **Note:** A [Firefox bug](https://bugzil.la/1647077) requires that the `loading` attribute must be placed before the `src` attribute, otherwise it has no effect.
```

>     > **Note:** A [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1647077) in Firefox requires that the **`loading` attribute must be placed before the `src` attribute**, otherwise it has no effect.

Reviewer 2: Could we use the shortlink version for this?

```suggestion
    > **Note:** A [Firefox bug 1647077](https://bugzil.la/1647077) requires that the **`loading` attribute must be placed before the `src` attribute**, otherwise it has no effect.
```

Reviewer 2: Fastest fingers first!

>     > **Note:** A [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1647077) in Firefox requires that the **`loading` attribute must be placed before the `src` attribute**, otherwise it has no effect.

Reviewer 3: ```suggestion
    > **Note:** A [bug](https://bugzil.la/1647077) in Firefox requires that the **`loading` attribute must be placed before the `src` attribute**, otherwise it has no effect.
```
